chart/epinio-ui/values.yaml

```diff
@@ -18,6 +18,9 @@ epinioApiUrl: http://epinio-server.epinio.svc.cluster.local
 # Skip checking for valid SSL cert when making requests to `EPINIO_API_URL`
 epinioApiSkipSSL: "true"
 
+# This is the version that is displayed in the ui and should match that of the epinio it's targetting
+epinioVersion: "0.5.0"
```

Comment:
I am bit confused by this setting. 
Could we be in a situation where the UI installed with Epinio 0.4?